cypress/integration/mentions.ts

```diff
@@ -0,0 +1,28 @@
+describe('mentions example', () => {
+  before(() => cy.visit('examples/mentions'))
+
+  it('renders mention element', () => {
+    cy.findByRole('textbox')
+      .get('[data-cy=mention-R2-D2')
+      .should('exist')
+      .get('[data-cy=mention-Mace-Windu]')
+      .should('exist')
+  })
+
+  it('shows list of mentions', () => {
+    cy.findByRole('textbox')
+      .type('{movetoend}')
+      .type(' @ma')
+      .get('[data-cy=mentions-portal]')
+      .should('exist')
+  })
+
+  it('insert on enter from list', () => {
```

Comment:
Just keeping the grammar consistent with other tests :)
```suggestion
  it('inserts on enter from list', () => {
```